unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -136,6 +136,19 @@ export function SuggestionsManagerTest() {
       expect(suggestion.getAttribute('aria-selected')).toBe('false');
     });
 
+    it('removes selected class and sets aria-selected to false when moving off a selected element in LockerService', () => {
+      suggestion.addClass(selectedClass);
+      suggestion.setAttribute('aria-selected', 'true');
+
+      suggestionManager.handleMouseOut({
+        target: suggestion.el,
+        relatedTarget: {}
```

Comment:
Small point, but you could make the test easier to understand if you stored the empty object in a well-named variable. e.g.

```
const lockerServiceRelatedTarget = {};
suggestionManager.handleMouseOut({
        target: suggestion.el,
        relatedTarget: lockerServiceRelatedTarget
});
```